crates/sui/src/keytool.rs

```diff
@@ -176,14 +176,14 @@ pub enum KeyToolCommand {
     /// This takes [enum SuiKeyPair] of Base64 encoded of 33-byte `flag || privkey`). It
```

Comment:
Need to change the comment to:
`/// This takes as input a Base64 private key`

Reply:
This comment still is valid since it needs to be a valid SuiKeyPair in base64 format

Reply:
Right! sorry about that.